config/routes.rb

```diff
@@ -68,6 +68,9 @@
       get '/chapter_selection', to: 'subscriptions#chapter_selection'
       get '/check_your_answers', to: 'subscriptions#check_your_answers' # for when user selects all chapters
       get '/subscription_confirmation', to: 'subscriptions#subscription_confirmation'
+      get '/unsubscribe_confirmation', to: 'subscriptions#unsubscribe_confirmation'
+      get '/unsubscribe', to: 'subscriptions#unsubscribe'
+      get '/unsubscribe_action', to: 'subscriptions#unsubscribe_action'
```

Comment:
I think this should be a post as it makes a change
```suggestion
      post '/unsubscribe', to: 'subscriptions#unsubscribe_action'
```

Reply:
@vertism I've updated to `POST` please re-review